base/friendly/friendly_test.go

```diff
@@ -9,22 +9,21 @@ import (
 )
 
 func TestFriendlyDiffDescriptions(t *testing.T) {
+	// TODO (b5) - we should update this test to use deepdiff to generate these diffs
```

Comment:
Disagree. I think it's correct to have unit tests that only exercise the code in friendly, in isolation from how deepdiff works. It makes it easier to see the expectations that the code makes, and how it behaves in isolated cases, helping users to understand the intentions of the code. We can also have additional tests higher up that verify the two in combination, as integration tests.

Reply:
I agree with your disagreement. Removing.